src/test/java/org/folio/fql/deserializer/FqlDeserializerEqualsTest.java

```diff
@@ -20,14 +20,33 @@ void setup() {
     fqlService = new FqlService();
   }
 
+  @Test
+  void shouldGetSimpleFqlWithStringValueAndVersion() {
+    String simpleStringJson =
+      """
+         {
+          "_version":123,
+          "fqlCondition":{
```

Comment:
er this doesn't look right...`_version` should go alongside `field1`. Maybe we do need to put the logic in FqlCondition's class?